package.json

```diff
@@ -2,6 +2,11 @@
   "name": "static-lite",
   "repository": "https://github.com/BrandonRomano/static-lite.git",
   "author": "Brandon Romano",
+  "contributors": {
+    "name": "Henry Snopek",
+    "email": "hhsnopek@gmail.com",
+    "url": "hhsnopek.com"
+  },
```

Comment:
Ahh you know what, this is totally going to bork all of my README line number references 😢 

Did you want to also update them?